_data/defis.yml

```
@@ -239,22 +237,57 @@
   image: /img/logo-lex-impact.png
   title: LexImpact
   link_to: /defis/2019/leximpact.html
+  source_code: https://github.com/betagouv/leximpact-server
   source_code: https://github.com/betagouv/leximpact-client
+  website: https://leximpact.beta.gouv.fr
   class: 2019 web design
   skills: EIG 3 // Web // Design
   colorback: red
-  description: "Evaluer l'impact des réformes socio-fiscales sur les citoyens"
+  description: "Permettre à nos parlementaires d'évaluer les impacts des réformes fiscales avant vote"
   year: 2019
   promotion: 3
   duration_months: 10
-  administration: "Direction interministérielle du numérique et du système d’information et de communication de l'État (DINSIC)"
+  administration: "Direction interministérielle du numérique (DINUM) et Assemblée nationale"
   administration_website: https://www.numerique.gouv.fr/
   eigs:
     - "Augustin Wenger"
     - "Dorine Lambinet"
   mentors:
-    - "Mauko Quiroga"
     - "Sandra Chakroun"
+    - "Mauko Quiroga"
+  key_figures:
+    - amount: 60
+      description: "secondes seulement pour estimer le coût d'un amendement, au lieu de plusieurs semaines"
+    - amount: 1
+      description: "La première base de données représentative de la population desormais accessible à l'usage par l'Assemblée nationale"
+    - amount: 40
+      description: "pourcent d'amendements concernés par l'outil ayant cité LexImpact dans l'exposé des motifs"
+    - amount: 84
+      description: "parlementaires et collaborateurs/collaboratrices ayant participé à une démonstration de l'outil"
+    - amount: 4
+      description: "discours en séance publique parlent de LexImpact"
+  end_of_project_videos:
+    - id: x6y04yt
+      platform: 
+      description: "10 mois après : présentation du défi LexImpact."
+      show: false
+  projects:
+    - image: /img/realisations/2019/LexImpact-PhotoInterface.jpg
+      title: "Le tableau de bord de LexImpact POP, accessibles aux parlementaires"
+      description: "Nous avons créé une interface facile d'utilisation permettant aux parlementaires d'estimer les impacts de leur amendement à l'Article 197 du CGI. En plus des impacts sur des cas types qu'ils peuvent configurer, ils ont une vision macro des effets sur la population et les recettes."
+    - image: 
+      title: "Estimer précisément les impacts sur les déciles de population"
+      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de "gagnants/perdants."
```

```suggestion
      description: "Nous avons pu proposer en temps et heure du Projet de loi de finances 2020, une visualisation précise des impacts, de l'impôt moyen payé par un foyer par déciles, au nombre de gagnants/perdants."
```

Tu ne peux pas avoir des guillemets doubles à l'intérieur du texte car sinon le YAML interprète ceci comme une fin d'un champ. Tu peux en revanche échapper un guillemet double `\"` si tu souhaites conserver des guillemets à l'intérieur.